.gitignore

```diff
@@ -28,7 +28,8 @@ src/mtev_version.h
 src/man/*.8
 src/man/*.h
 src/Makefile.objs
-src/mtev_config.h
+src/config.h
+src/config.h.in
 src/scripts/mtev-config
```

Comment:
The new separation of mtev_config.h into that and config.h will cause problems.  We ship mtev_config.h and it is namespaced sufficiently, but now depends upon config.h

It looks like the intended separation is to keep autoconfy things separate.  That was the original intention behind mtev_config.h and mtev_defines.h.

mtev_config.h.in should stay and mtev_defines.h should adopt the things in that were put in there as a part of this proposal.